src/app/components/Stats/Stats.tsx

```diff
@@ -135,7 +137,14 @@ export const Stats: React.FC = () => {
       {
         title: "Confirmed TVL",
         value: stakingStats?.activeTVLSat
-          ? `${maxDecimals(satoshiToBtc(stakingStats.activeTVLSat), 2)} ${coinName}`
+          ? `${maxDecimals(satoshiToBtc(stakingStats.activeTVLSat), 2)} ${coinName}${
+              stakingStats.btcPriceUsd
```

Comment:
On testnet this should always be `$0`

Reply:
I don't mind not showing at all. What's important is that we do not show any real dollar values next to test tokens.

Reply:
understood, I will merge this PR as is for now. In the public facing testnet we will not add the `btcPriceUsd` in the config from the api side cc:@filippos47